src/UI/Component/Panel/Listing/Listing.php

```diff
@@ -22,4 +22,18 @@ public function getTitle();
 	 * @return \ILIAS\UI\Component\Item\Group[]
 	 */
 	public function getItemGroups();
+
+	/**
+	 * Sets the action drop down to be displayed on the right of the title
+	 * @param \ILIAS\UI\Component\Dropdown\Standard $actions
+	 * @return Listing
+	 */
+	public function withActions(\ILIAS\UI\Component\Dropdown\Standard $actions);
```

Comment:
Consumers need to be allowed to set the label. The calendar sets this to "Days" which is not a general label. To me this is the same pattern as already being implemented for items and item groups.